### PR TITLE
Update mac-jamfpro-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the navigation for Application & Custom Settings on Jamf pro 10.33.0

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6949

File: microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ You'll need to take the following steps:
    - Distribution Method: Install Automatically
    - Level: Computer Level
 
-4. In **Application & Custom Settings** select **Configure**.
+4.  Navigate to the **Application & Custom Settings** payload and select **Upload** > **Add**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md]:
```suggestion
4.  Navigate to the **Application & Custom Settings** page and select **Upload** > **Add**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**